content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,109 @@
+
+Okay, server time then! This time we focused heavily in migrating to .NET Core 3.0 (the framework we use for building the server's core).
+Although this seems boring, let's see some benchmarks from the hex decoder (used in server's tasks):
+
+```
+|                         Method |     Mean |    Error |   StdDev |   Gen 0 | Gen 1 | Gen 2 | Allocated |
+|------------------------------- |---------:|---------:|---------:|--------:|------:|------:|----------:|
+|                New hex decoder | 10.81 us | 0.079 us | 0.074 us |  0.1068 |     - |     - |     336 B |
+|                Old hex decoder | 23.57 us | 0.055 us | 0.051 us | 11.2305 |     - |     - |   35272 B |
+```
+*Shoutout to [Bond-009](https://github.com/Bond-009) for this!*
+
+That's only a tiny example of what we have been cooking and the groundwork preparation we've been doing for upcoming versions.
+However, there are some key improvements here, which are mostly focused in performance:
+
+* Added support for embedded attachments in media files (#1838)
+* Fixed Play To order for playlists
+* Raspbery Pi should have now full hardware-acceleration capabilities when encoding and decoding (older models, not in 4 yet)
+* General improvements to the transcoding speed and reliability
+* Reliability improvements
+* Tons of bug fixes
+
+## Release Notes
+
+All this blog post focused heavily in the server and the web client, as they're the most used ones. However, we also have a bunch of apps that will receive
+this update as well, so [keep an eye](https://github.com/jellyfin) on our GitHub.
+
+Detailed release notes available on the [GitHub's server repository](https://github.com/jellyfin/jellyfin/releases/tag/v10.5.0).
+
+## We're not done yet
+
+As [Joshua explained in his blog post](https://jellyfin.org/posts/jellyfin-in-2019/), we're really excited to see how our whole community is evolving,
+and we're impressed by how far we got since we first started. Things are going really quick, and this release just proves that.
+
+In this post, I mentioned some of the people that contributed a lot of making this version possible. However, it's completely unfair
+if only they're mentioned, as this has been a community work. That's why I want to take advantage of this blog post to say thank you, in behalf of all the project
+leaders, to every single one of you out there: thank you very much for the sponsoring, thank you very much for the code contributions, bug reporting, testing...
+Thank you very much to all our translators, reddit/matrix/forum users. I also want to say a big *Thank You* to (also) a majority of users, those who simply download this software and never need support, neither suits their needs and never came back; also thank you to you who (might) be reading this and you're hearing about us right now: the solely reason of taking some time to take a look at what we do means that you care.
```

Comment:
As the first sentence is covered in the last review, the rest is covered here.

"I also want to say a big thank you to the users who have simply downloaded JF regardless of it fitting your needs. We hope that you will stick around and see what else we have in store for the future!"

Instead of the whole mass of commands, colons, and semicolons you have there now.

Reply:
Also fixed in the new commit, you reviewed some old parts :)